data/json/recipes/electronics/parts.json

```diff
@@ -119,6 +119,22 @@
     "qualities": [ { "id": "SCREW", "level": 1 } ],
     "components": [ [ [ "scrap", 6 ] ], [ [ "cable", 4 ] ], [ [ "plut_cell", 20 ] ] ]
   },
+  {
+    "result": "flux_comp_gen",
+    "type": "recipe",
+    "category": "CC_ELECTRONIC",
+    "subcategory": "CSC_ELECTRONIC_PARTS",
+    "skill_used": "electronics",
+    "difficulty": 5,
+    "skills_required": [ "fabrication", 4 ],
+    "time": "60 m",
+    "decomp_learn": 4,
+    "book_learn": [ [ "textbook_anarch", 5 ], [ "advanced_electronics", 5 ] ],
+    "using": [ [ "soldering_standard", 10 ] ],
+    "qualities": [ { "id": "SCREW", "level": 1 } ],
+    "tools": [ [ [ "crucible", -1 ], [ "crucible_clay", -1 ] ], [ [ "forge", 25 ] ] ],
+    "components": [ [ [ "cu_pipe", 2 ] ], [ [ "cable", 241 ] ], [ [ "composite_b", 450 ] ], [ [ "concrete", 1 ] ] ]
```

Comment:
In order to actually make something from concrete, I think you'd also need a concrete mixer, some water and a concrete smoothing tool (currently existing as SMOOTH quality; 1 should do); also you may want to replace forge tool with currently existing "forging_standard" requierement.

Reply:
mixing the concrete by hand should be doable since it is only ~1.5 kg. good call on the smoothing, water, and forging_standard. Adding them now